README.md

```diff
@@ -1,9 +1,9 @@
 # Avanti Fellows OTP Service
 
-This repository contains code for sending, re-sending and verifying OTP sent to mobile numbers.
+This repository contains code for sending and verifying OTP sent to mobile numbers.
 
 ## Usage
-The service is exposed in the form of three endpoints
+The service is exposed in the form of two endpoints
```

Comment:
as you said in your description, this is really one endpoint with different functionalities based on the params, right? We should write that here.

Reply:
Our API has two endpoints, the Gupshup's API has only one.